src/site/content/en/lighthouse-accessibility/aria-required-children/index.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: Ensure parent roles contain required child role(s)
+description: |
+  Learn about aria-required-children audit.
+author: megginkearney
+web_lighthouse:
+  - aria-required-children
+---
+
+[ARIA](https://www.w3.org/TR/wai-aria-1.1/#role_definitions)
+roles and attributes help screen readers
+provide missing information about an element.
+For these roles and attributles to make sense,
+each ARIA `role` supports a specific subset of `aria-*` attributes
+(see [ARIA roles definitions](https://www.w3.org/TR/wai-aria-1.1/#role_definitions)).
+Some ARIA `roles` require child roles.
+Lighthouse reports when a role is missing their required child role(s):
+
+<figure class="w-figure">
+  <img class="w-screenshot w-screenshot--filled" src="aria-required-children.png" alt="Lighthouse: ARIA role missing required child role(s)">
```

Comment:
I think a different image should be used here. The `aria-checked` is kind of a red herring, unrelated to the issue.

Reply:
I've got a new screen shot.